client/src/helpers/authFetch.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Create a wrapper function that add the token as config into fetch() function
+ */
+
+export const authFetch = (...args) => {
+    let token = localStorage.getItem('access_token') || null
+    let config = {}
+
+    if(token) {
+        config = {
+            headers: { 'Authorization': `Bearer ${token}` }
+        }
```

Comment:
This doesn't look right to me, since the method, body, etc would go in the config